test/InterchainToken.js

```diff
@@ -147,7 +147,7 @@ describe('InterchainToken', () => {
             const contractBytecodeHash = keccak256(contractBytecode);
 
             const expected = {
-                london: '0xa01cf28b0b6ce6dc3b466e995585d69486400d671fce0ea8d06beba583e6f3bb',
+                london: '0x482146829055f052063003e9cf0ffaf798a12fb58088c2667566a135b9568355',
```

Comment:
Avoid changing contracts with fixed bytecode unless necessary. We want to avoid multiple versions of tokens to be active at the same time since we can't upgrade previous ones